README.md

```diff
@@ -17,26 +23,191 @@ The SDK handles the device data collection, communication with the card issuer,
+  pod 'JOSESwift', '2.2.1', :inhibit_warnings => true
+  pod 'CheckoutEventLoggerKit', '1.2.3'
+  pod 'Checkout3DS', :git => 'git@github.com:checkout/checkout-3ds-sdk-ios.git', :tag => '3.0.0'
+end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['BUILD_LIBRARY_FOR_DISTRIBUTION'] = 'YES'
+    end
+  end
+end
+
+```
+
+Then, run the following command in terminal:
+
+```bash
+$ pod install
+```
+Note: your Pod setup may have the following [issue](https://github.com/checkout/checkout-3ds-sdk-ios/issues/4)
+
+To update your existing Cocoapod dependencies, use:
+```bash
+$ pod update
+```
+
+
 Then, configure your app to:
 
-1. Initialize the SDK with your preferred user interface options.
+### Integrating with Checkout 3DS servers
+This integration method involves one call to our `authenticate` method, which will perform an entire Authentication flow using Checkout.com's 3DS servers.
+
+1. Initialize the SDK with your preferred user interface options using our `uiCustomization` object.
 2. Configure the parameters for an authentication.
 3. Request authentication and handle the result to continue your payment flow.
 
-
+#### Code snippet
+
+```swift
+// 1. Init with defaults
+let checkout3DS = Checkout3DSService()
+
+// 2. Init with explicit arguments
+let checkout3DS = Checkout3DSService(
+    environment: .production,
+    locale: Locale(identifier: "en_GB"),
+    uiCustomization: uiCustomization,
+    appURL: URL(string: "myapp://my-app-url")!
+)
+
+let authenticationParameters = AuthenticationParameters(
+    sessionID: sessionID,
+    sessionSecret: sessionSecret,
+    scheme: scheme)
+
+checkout3DS.authenticate(authenticationParameters: authenticationParameters) { error in
+    if let error = error {
+        // Handle error.
+    } else {
+        // Continue with payment.
+    }
+}
+```
 👉&nbsp;&nbsp;[See the Integration Guide with Code Examples](https://www.checkout.com/docs/risk-management/3d-secure/sessions/non-hosted-sessions/3d-secure-mobile-sdks) for full details.
 
 📚&nbsp;&nbsp;[Read the reference documentation](https://checkout.github.io/checkout-mobile-docs/checkout-3ds-sdk-ios/index.html)
 
+### Integrating with any 3DS provider
+Standalone 3DS allows our SDKs to be used with any Authentication provider, be it Checkout.com or otherwise. This is a higher touch integration that breaks up the 3DS flow more granularly. In order to integrate with the Standalone 3DS service:
+
+1. Initialize the SDK with your preferred user interface options using our `uiCustomization` object.
+2. Create `transaction` object 
+3. Get `authenticationRequestParameters` for the AReq
+4. if a challenge is mandated from the Authentication response from your 3DS server then call the `doChallenge` method to render the challenge however, if the challenge is not mandated by the ACS then it would have triggered a frictionless 3DS flow. 
```

Comment:
```suggestion
4. If a challenge is mandated from the Authentication response from your 3DS server then call the `doChallenge` method to render the challenge however, if the challenge is not mandated by the ACS then it would have triggered a frictionless 3DS flow. 
```